<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Black-Webbrowser Setting</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
    <widget class="QComboBox" name="themes">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Black-Webbrowser v2.0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="font_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Font</string>
     </property>
    </widget>
    <widget class="QPushButton" name="license_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>230</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>License</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_data_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="history_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bookmark_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Bookmark</string>
     </property>
    </widget>
    <widget class="QPushButton" name="download_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Downloads</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="source_btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>310</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Source</string>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="about_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_feedback">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Feedback</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Home Page</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="home_line">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Url...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ok_p_btn">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_line">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Username...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ok_p_btn_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set User Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="instagram_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Instagram</string>
     </property>
    </widget>
    <widget class="QPushButton" name="twitter_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Twitter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="github_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Github</string>
     </property>
    </widget>
    <widget class="QPushButton" name="telegram_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Telegram</string>
     </property>
    </widget>
    <widget class="QPushButton" name="facebook_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>FaceBook</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_emailbtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Email To Developer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fullscreen_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fullscreen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="normal_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minimize_btn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimize</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
